samples/drivers/touch/boards/native_posix_64.overlay

```diff
@@ -0,0 +1,6 @@
+/ {
+	chosen {
+		zephyr,display = &sdl_dc;
+		zephyr,touch = &input_sdl_touch;
```

Comment:
the first is already in `boards/native/native_sim/native_sim.dts`, it should not be needed here, the second should probably go in the board file, @kartben what do you think?